installation-guide.md

```diff
@@ -0,0 +1,143 @@
+# iOS AEP SDKs installation guide
+
+## Migration considerations 
+
+Here are a few things to consider when installing the Adobe Experience Platform Edge Network mobile extension.
+
+- The new AEP Edge Network extension requires the usage of the latest AEP SDKs, which for iOS is a new Swift-based SDK. If you are an ACP Mobile SDK user, read more about this in the [Adobe Experience Platform Edge Network Migration Considerations](../getting-started/introduction-to-aepedge.md#adobe-experience-platform-edge-network-migration-considerations).
```

Comment:
I think you can remove the "more about this in the" and just say "If you are an ACP Mobile SDK user, read Adobe Experience Platform Edge Network Migration Considerations."